MoneySplitter.Models/TransactionModel.cs

```diff
@@ -1,7 +1,8 @@
 ﻿using System;
-
-namespace MoneySplitter.Models.Entities
+using System.Runtime.Serialization;
+namespace MoneySplitter.Models
 {
+    [DataContract]
```

Comment:
Models for Api in .Models project again?

Reply:
Add defines class